templates/Rankings.html

```diff
@@ -0,0 +1,48 @@
+{% extends 'templates/base.html' %}
```

Comment:
Is this coming from Jinja? Bottle uses a different template engine called [SimpleTemplateEngine](https://bottlepy.org/docs/dev/stpl.html#stpl.rebase). In case you want to use Jinja for templates make sure you install that properly.

If not, use the simple template syntax.

Reply:
I don't know.Actually I didn't understood Simple Template Engine documentation properly so I just googled it and got a tutorial by Djnago Girls and there it was not mentioned. I think maybe `from bottle import SimpleTemplate` we need to use this instead of `{% extends 'templates/base.html' %}` but I am not sure.